rustls/src/server/server_conn.rs

```diff
@@ -251,6 +260,39 @@ impl ServerConfig {
     }
 }
 
+/// A structure which allows reading of early data (also known as
```

Comment:
For best results in rustdoc, the first sentence of a docstring should fit on one line.